packages/bot-arbitrage/src/uniswap/conversion.ts

```diff
@@ -0,0 +1,20 @@
+import JSBI from "jsbi";
```

Comment:
Since I don't know which version of node you can't we use native BigInt ?

Reply:
This is taken directly from Uniswaps own tutorial. I did not spend much time in converting to our own why of handling big numbers